docs/synapse/devopsguides/ingest_intro.rst

```diff
@@ -0,0 +1,641 @@
+    inet:netuser = socialnetwork.ninja/bobtheuser
+    inet:netuser = socialnetwork.ninja/mallorythespy
+    (3 results)
+    cli> ask --props inet:netuser=socialnetwork.ninja/bobtheuser inet:netuser->inet:follows:follower show:cols(inet:follows:follower, inet:follows:followee)
+    socialnetwork.ninja/bobtheuser  socialnetwork.ninja/mallorythespy
+    socialnetwork.ninja/bobtheuser socialnetwork.ninja/alicethefriend
+    (2 results)
+    cli> ask --props inet:netuser=socialnetwork.ninja/bobtheuser inet:netuser->inet:netpost:netuser
+    inet:netpost = 0e89cf5db1fd8c426daa8b01e58cd2dd
+        :netuser = socialnetwork.ninja/bobtheuser
+        :netuser:site = socialnetwork.ninja
+        :netuser:user = bobtheuser
+        :text = hallo!
+        :time = 2017/05/02 01:06:03.000
+    inet:netpost = 16a550ca28e7e62e0b47b482cc02b58e
+        :netuser = socialnetwork.ninja/bobtheuser
+        :netuser:site = socialnetwork.ninja
+        :netuser:user = bobtheuser
+        :text = Just got back from the concert
+     had a great time
+        :time = 2017/08/17 01:02:03.000
+    inet:netpost = e6b5d03f398a02548d9201efbdc58a06
+        :netuser = socialnetwork.ninja/bobtheuser
+        :netuser:site = socialnetwork.ninja
+        :netuser:user = bobtheuser
+        :text = Just did hax with nine bit bytes @ defcon
+        :time = 2017/08/01 02:03:04.000
+    (3 results)
+
+When ingesting data in this format, it is common to also apply tags to a ingest definition. This can be useful
+for adding additional analytical data or categorization to the nodes as they are created in the Cortex. This can be done
+by adding a "tags" directive in line with other ingest directives.  An example of modifying the above ingest to add a
+``#src.socialnetwork`` tag can be seen highlighted below:
+
+.. literalinclude:: examples/ingest_structured_nested_def2.json
+    :language: json
+    :emphasize-lines: 11-13
+
+This will add the ``#src.socialnetwork`` tag to all of the nodes created directly by the ingest process. This ingest
+can be run via the ingest tool::
+
+    python -m synapse.tools.ingest --verbose  --core sqlite:///embed_examples.db  docs/synapse/devopsguides/examples/ingest_structured_nested_def2.json
+
+Then we can lift nodes from this ingest via tags. The following example shows lifting all of the ``inet:netuser`` nodes
+made with this ingest::
+
+    ~/synapse$ python -m synapse.cortex sqlite:///embed_examples.db
+    cli> ask inet:netuser*tag=src.socialnetwork
+    inet:netuser = socialnetwork.ninja/bobtheuser
+        #src.socialnetwork (added 2017/08/17 23:53:30.024)
+    (1 results)
+
+This result may seem odd - in the previous query ``ask inet:netuser:site=socialnetwork.ninja`` we had three
+``inet:netuser`` accounts found; but ``ask inet:netuser*tag=src.socialnetwork`` only gave us one account. This is
+because we only created one ``inet:netuser`` account directly in the ingest definition, the other accounts were created
```

Comment:
the nodes are created but not tagged - we only tag nodes added directly in the ingest definition.